.github/workflows/build.yml

```diff
@@ -116,7 +116,31 @@ jobs:
         run: |
           valgrind --leak-check=full --leak-check-heuristics=all --undef-value-errors=no --error-exitcode=9 ./bin/minetest --run-unittests
 
-          
+
+  clang_9_prometheus:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install compiler
+        run: |
+          sudo apt-get install clang-9 -qyy
+          source ./util/ci/common.sh
+          install_linux_deps
+
+      - name: Build prometheus-cpp
+        run: |
+          ./util/ci/build_prometheus_cpp.sh
```

Comment:
Since the docker build already tests prometheus, is this necessary?

Reply:
it's not the same context, one is alpine this one is ubuntu, the regular one

Reply:
I know. But this is all extra work/time that the CI has to do.
I'd consider it unlikely that Prometheus works on Alpine but is broken on Ubuntu at the same time.

Reply:
i think as it's not the same build context, this one is in ubuntu, the image is built in alpine